pystac_client/client.py

```diff
@@ -142,7 +142,7 @@ def get_all_items(self) -> Iterable["Item_Type"]:
         else:
             yield from super().get_items()
 
-    def search(self, require_geojson_link: bool = False, **kwargs: Any) -> ItemSearch:
```

Comment:
this was unused, and there was no note about it capturing the kwarg so it wouldn't be passed on.